### ISSUE TITLE
Will librocksdb-sys work on nightly in the future?

### ISSUE THREAD

**Issue by rw** [rust-rocksdb/rust-rocksdb#341]
Following up from issue https://github.com/rust-rocksdb/rust-rocksdb/issues/335, I need to know if rocksdb will start working again on nightly at some point. Thanks!

**Comment by aleksuss** [comment]
The next release will be working. I believe.

**Comment by rw** [comment]
@aleksuss Would you recommend that I use the `master` branch in the meantime?

**Comment by aleksuss** [comment]
@rw Yes. I would. But you should consider that the master branch has breaking changes already.

**Comment by rw** [comment]
@aleksuss Breaking API changes? No big deal! Those are an improvement over a library that doesn't compile  :-]

**Comment by YaronWittenstein** [comment]
@aleksuss do you have any ETA for the next release?
Thanks!

**Comment by aleksuss** [comment]
@rw could you please close the issue ?
